resources/integration_test/tests/query.yaml

```
@@ -0,0 +1,197 @@
+tests:
+  - description: select_star_from_grades_ordered
+    db: integration_test
+    sql: SELECT * FROM grades ORDER BY _id
+    ordered: true
+    skip_reason: SQL-659
+
+  - description: select_star_from_class_unordered
+    db: integration_test
+    sql: SELECT * FROM class
+    expected_result:
+      - [ 0, true, John, 2000-01-01, 10329 ]
+      - [ 1, false, Jane, 2000-02-01, 342 ]
+      - [ 2, true, Mike, 2000-01-01, 303 ]
+      - [ 3, false, Mary, 2000-03-01, 204323 ]
+      - [ 4, false, Pete, 2000-01-01, 10 ]
+    row_count: 5
+    ordered: false
+    expected_sql_type: [ INTEGER, BIT, LONGVARCHAR, TIMESTAMP, INTEGER ]
+    expected_catalog_name: [ '', '', '', '', '' ]
+    expected_column_class_name: [ int, boolean, java.lang.String, java.sql.Timestamp,
+                                  int ]
+    expected_column_label: [ _id, enrolled, name, startdate, studentid ]
+    expected_column_display_size: [ 10, 1, 0, 24, 10 ]
+    expected_precision: [ 10, 1, 0, 24, 10 ]
+    expected_scale: [ 0, 0, 0, 0, 0 ]
+    expected_schema_name: [ '', '', '', '', '' ]
+    expected_is_auto_increment: [ false, false, false, false, false ]
+    expected_is_case_sensitive: [ false, false, true, false, false ]
+    expected_is_currency: [ false, false, false, false, false ]
+    expected_is_definitely_writable: [ false, false, false, false, false ]
+    expected_is_nullable: [ columnNullable, columnNullable, columnNullable, columnNullable,
+                            columnNullable ]
+    expected_is_read_only: [ true, true, true, true, true ]
+    expected_is_searchable: [ true, true, true, true, true ]
+    expected_is_signed: [ true, false, false, false, true ]
+    expected_is_writable: [ false, false, false, false, false ]
+
+  - description: select_as_confirm_column_label
+    db: integration_test
+    sql: SELECT enrolled AS e, studentid AS s, name AS n from class
```

```suggestion
    sql: SELECT enrolled AS e, studentid AS s, name AS n FROM class
```
[super-nit] looks like we've typically CAPS the SQL keywords but we missed "from" -> "FROM" here!